src/Layout.js

```diff
@@ -1068,8 +1068,32 @@ var computeLayout = (function() {
       child.nextAbsoluteChild = null;
     }
   }
+  
+  function layoutNode(node, parentMaxWidth, parentDirection) {
+    if (!node.lastLayout ||
+         node.lastLayout.requestedHeight !== node.layout.height ||
+         node.lastLayout.requestedWidth !== node.layout.width ||
+         node.lastLayout.parentMaxWidth !== parentMaxWidth) {
+      
+      if (!node.lastLayout)
+        node.lastLayout = {};
+      
+      node.lastLayout.requestedWidth = node.layout.width;
+      node.lastLayout.requestedHeight = node.layout.height;
+      node.lastLayout.parentMaxWidth = parentMaxWidth;
+    
+      layoutNodeImpl(node, parentMaxWidth, parentDirection);
+      
+      for (var key in node.layout)
```

Comment:
@vjeux - this should probably be added to the static code analysis as part of the CI build. I'll look into it.